app/uk/gov/nationalarchives/omega/editorial/models/package.scala

```diff
@@ -133,3 +136,15 @@ package object editSets {
   def getEditSet(): EditSet =
     editSet1
 }
+
+package object legalStatus {
+  val legalStatusData =
+    Seq(
+      LegalStatus("ref.0", "Select a Legal Status"),
```

Comment:
This makes it look like `ref.0` is a legitimate legal service; instead I would remove this element, and add the initial element in the view itself, with a value of `""`.

Reply:
I agree. This data should represent what will be received from the services API and that will not have anything with this text in. This should be created as a distinct `SelectItem` in the view and it should marked as disabled so that it cannot be selected.

Reply:
Good idea about the disabling it; I'll do the same for [this similar story](https://national-archives.atlassian.net/browse/PACT-607).

Reply:
I believe we've agreed to postpone `disabled` for now.